pgrx-pg-sys/src/pg12.rs

```diff
@@ -740,9 +741,9 @@ pub const LC_IDENTIFICATION_MASK: u32 = 4096;
 pub const LC_ALL_MASK: u32 = 8127;
 pub const HAVE_PG_ATTRIBUTE_NORETURN: u32 = 1;
 pub const HAVE_PRAGMA_GCC_SYSTEM_HEADER: u32 = 1;
+pub const __bool_true_false_are_defined: u32 = 1;
 pub const true_: u32 = 1;
 pub const false_: u32 = 0;
-pub const __bool_true_false_are_defined: u32 = 1;
```

Comment:
did the sorting change...???

Reply:
must have.  This happens on almost every release.  Or I've updated to the latest pg sources and then the bindings diff is even bigger.